pkg/chain/ethereum/tbtc.go

```diff
@@ -1126,6 +1186,37 @@ func (tc *TbtcChain) GetDepositRequest(
 	}, nil
 }
 
+func (tc *TbtcChain) Deposits(
```

Comment:
Note to self: this should be removed after #3654.

Reply:
Removed in cbd754abb9a1718e495209884eb4714db6e5ad68.